src/utils/API.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const API = axios.create({
+  baseURL: "https://gotogro-msp20-api-test.herokuapp.com",
+  // baseURL: "http://localhost:4000",
+  responseType: "json",
+});
+
+export const addMember = async (memberToAdd, successModalRef, errorModalRef) => {
+  try {
+    const response = await API.post("/addmember", {
+      lastName: memberToAdd.lastName,
+      firstName: memberToAdd.firstName,
+      dob: memberToAdd.dob,
+      email: memberToAdd.email,
+      mobile: memberToAdd.mobile,
+    });
+    successModalRef.current.showModal();
```

Comment:
- If the server returns a success response, open the success popup
- If the server returns an error response, open the error popup